tendrl/ceph_integration/sds_sync/__init__.py

```diff
@@ -82,6 +89,48 @@ def on_heartbeat(self, cluster_data):
             if data:
                 self.on_sync_object(data)
 
+        # Get and update ec profiles for the cluster
+        self._sync_ec_profiles()
+
+    def _sync_ec_profiles(self):
```

Comment:
I think this CLI parsing stuff we need to avoid. Is there a way to get this through any of the maps? or through rados?
same is the for rbd